### PR TITLE
Development Containers

### PR DESCRIPTION
**Pull request by alexyao2015**

### CODE REVIEW

File: emulated_hue/config.py [hunk 1]
```diff
@@ -16,7 +16,7 @@
 class Config:
     """Hold configuration variables for the emulated hue bridge."""
 
-    def __init__(self, hue, data_path, hass_url, hass_token):
+    def __init__(self, hue, data_path, hass_url, hass_token, ip_address, mac_address):
```
Comment (marcelveldt) [inline emulated_hue/config.py:19]: Maybe have these as optional params by setting default value to None.
In the check below, use a check for None to use the previous call to get the ip

**Review by marcelveldt — CHANGES_REQUESTED** [review]
Looking good!
Just one small comment

File: emulated_hue/config.py (hunk 1 above)
Comment (alexyao2015) [reply emulated_hue/config.py:19]: Hmm yes good idea. Although they are currently "optional" as they are defaulted to None by the argument parser. What do you think is better? Feel free to commit changes.

Comment (marcelveldt) [reply emulated_hue/config.py:19]: The parses is not used when we're called as library, only when called as script (__main__.py) so best practice to make sure the API doesn't break.

Comment (alexyao2015) [reply emulated_hue/config.py:19]: Ah I guess I didn't see that it would be used as a library, but I suppose I could add that. Doesn't hurt.

**Pull request closed without merging**